tests/utils.py

```diff
@@ -507,6 +507,28 @@ def subd_pid(self, subd):
                 return group.group(1)
         raise ValueError("No daemon {} found".format(subd))
 
+    def channel_state(self, other):
+        """Return the state of the channel to the other node.
+
+        Returns None if there is no such peer, or a channel hasn't been funded
+        yet.
+
+        """
+        peers = self.rpc.listpeers(other.info['id'])['peers']
+        if not peers or 'channels' not in peers[0]:
+            return None
+        channel = peers[0]['channels'][0]
```

Comment:
There can be multiple channels (ie. onchain ones, too).  Use only_one(peers[0]['channels']) here, since we're assuming there's only one?